packages/twenty-front/src/modules/object-record/record-board/record-board-column/hooks/useAddNewOpportunity.ts

```diff
@@ -47,11 +47,19 @@ export const useAddNewOpportunity = (position: string) => {
   );
 
   const handleAddNewOpportunityClick = useCallback(() => {
-    setIsCreatingCard(true);
+    if (isDefined(isOpportunitiesCompanyFieldDisabled)) {
```

Comment:
wrong condition here. isDefined(false) returns `true`
```suggestion
    if (isOpportunitiesCompanyFieldDisabled) {
```